xbmc/addons/interfaces/GUI/Window.cpp

```diff
@@ -280,7 +315,281 @@ bool Interface_GUIWindow::do_modal(void* kodiBase, void* handle)
 
   return true;
 }
+//@}
+
+/*!
+ * Window control functions
+ */
+//@{
+bool Interface_GUIWindow::set_focus_id(void* kodiBase, void* handle, int control_id)
+{
+  CAddonDll* addon = static_cast<CAddonDll*>(kodiBase);
+  CGUIAddonWindow* pAddonWindow = static_cast<CGUIAddonWindow*>(handle);
```

Comment:
Whenever you cast a void to something else, imo, reinterpret_cast is the best choice.